dojo/settings/settings.dist.py

```diff
@@ -326,6 +326,8 @@
     # For HTTP requests, how long connection is open before timeout
     # This settings apply only on requests performed by "requests" lib used in Dojo code (if some included lib is using "requests" as well, this does not apply there)
     DD_REQUESTS_TIMEOUT=(int, 30),
+    # Dictates if v3 org/asset relabeling (+url routing) will be enabled
+    DD_ENABLE_V3_ORGANIZATION_ASSET_RELABEL=(bool, True),
```

Comment:
Let's start with this off for now, and then we can cut over in the near future 
```suggestion
    DD_ENABLE_V3_ORGANIZATION_ASSET_RELABEL=(bool, False),
```